style.css

```diff
@@ -672,56 +671,49 @@ figcaption.wp-caption-text {
 }
 
 .sss {
+	display: block;
 	height: 0;
-	margin: 0;
-	padding: 0;
 	position: relative;
-	display: block;
 }
 
 .sss .ssslide {
-	width: 100%;
-	margin: 0 auto;
+	display: none;
+	overflow: hidden;
 	padding: 0;
 	position: absolute;
 	top: 0;
-	left: 0;
-	right: 0;
-	display: none;
-	overflow: hidden;
+	width: 100%;
 }
 
-.ssslide img {
-	max-width: 100%;
-	height: auto;
-	margin: 0;
-	padding: 0;
-	position: relative;
-	display: block;
+.ssslide-inner {
+	display: table;
+	margin: 0 auto;
+}
+
+.ssslide figcaption {
+	caption-side: bottom;
+	display: table-caption;
+	width: auto;
 }
 
-.sssnext, .sssprev {
-	width: 25px;
+.sssnext,
+.sssprev {
+	background: center no-repeat;
+	cursor: pointer;
 	height: 100%;
-	margin: 0;
 	position: absolute;
 	top: 0;
+	width: 1.8rem;
 }
 
 .sssprev {
 	left: -1.8rem;
-	background: url('https://magazine.wsu.edu/wp-content/themes/wsu-magazine-theme/images/left-arrow_icon.svg') no-repeat;
-	background-position: 10px;
+	background-image: url('https://magazine.wsu.edu/wp-content/themes/wsu-magazine-theme/images/left-arrow_icon.svg');
```

Comment:
Let's change these background URLs to be relative - `url("./images/left-arrow_icon.svg");`

We'll be making some Amazon S3 related changes with uploaded content and future-proofing this would be nice.